docs/src/landing.md

```diff
@@ -0,0 +1,59 @@
+# Welcome to the `ethstaker-deposit-cli`!
+
+The [`ethstaker-deposit-cli`](https://github.com/eth-educators/ethstaker-deposit-cli) is a command-line tool forked from Ethereum's [`staking-deposit-cli`](https://github.com/ethereum/staking-deposit-cli) with more functionality and improved performance.
+
+## Important Concerns
+
+This tool generates and/or utilizes mnemonics and validator keystores. Both are highly sensitive materials and their exposure could lead to the loss or theft of funds. Users must take appropriate measures to secure this information.
+
+Here are some core recommendations:
+
+- **Use this tool offline**: We highly recommend using this tool in an offline environment, preferably air-gapped, using a live USB such as [Tails](https://tails.net/install/download/index.en.html).
+
+- **Avoid using passwords or mnemonics as command line arguments**: When executing any command, you will be prompted for the necessary arguments. This prevents the arguments from being stored in your command history which could create an attack vector.
```

Comment:
I just noticed you mention avoiding some CLI arguments here related to security. Even if we mention it here, it should also be mentioned immedialy next to actual arguments in each command page (inline or maybe with an [mdBook warning section](https://rust-lang.github.io/mdBook/format/mdbook.html#classwarning))